pyat/at/physics/rdt.py

```diff
@@ -0,0 +1,545 @@
+from __future__ import annotations
+
+import numpy
+import multiprocessing
+from functools import partial
+from ..lattice import Lattice, Refpts, Multipole, All
+from enum import Enum
+from collections.abc import Sequence
+from dataclasses import dataclass, asdict
+
+
+__all__ = ["get_rdts", "RDTType"]
+
+
+class RDTType(Enum):
+    """Enum class for RDT type"""
+
+    ALL = 0
+    FOCUSING = 1
+    COUPLING = 2
+    CHROMATIC = 3
+    GEOMETRIC1 = 4
+    GEOMETRIC2 = 5
+    TUNESHIFT = 6
+
+
+@dataclass
+class _RDT:
+    # Location
+    refpts: Sequence[int] | int = None
+    # Normal quadrupoles rdts
+    h20000: Sequence[complex] | complex = None
+    h00200: Sequence[complex] | complex = None
```

Comment:
h00200 is missing in the matlab implementation. Why is it left out ?

Reply:
They are missing in the elegant implementation, that's why they are missing in the Matlab implementation!